Build/Tactical/Soldier_Create.cc

```diff
@@ -1254,6 +1255,8 @@ void CreateDetailedPlacementGivenBasicPlacementInfo( SOLDIERCREATE_STRUCT *pp, B
 			break;
 		case SOLDIER_CLASS_ELITE:
 			pp->bExpLevel = (INT8) 6 + bExpLevelModifier;
+			if(pp->bExpLevel<GCM->getGamePolicy()->enemy_elite_minimum_level) pp->bExpLevel=GCM->getGamePolicy()->enemy_elite_minimum_level;
```

Comment:
No, it's just bad style. In one of the following commits I cleaned it up (the abstraction one).